core/controllers/admin.py

```diff
@@ -128,9 +123,9 @@ def get(self):
 
         self.values.update({
             'continuous_computations_data': continuous_computations_data,
-            'demo_collections': feconf.DEMO_COLLECTIONS.iteritems(),
+            'demo_collections': sorted(feconf.DEMO_COLLECTIONS.iteritems()),
             'demo_collection_ids': demo_collection_ids,
```

Comment:
``demo_exploration_ids`` is used by the *Reload Explorations* button. ``demo_collection_ids`` is there for the future *Reload Collections* button, but I agree that it probably can be removed and added in later if we have more than one collection to reload.